cuckoo/processing/memory.py

```diff
@@ -938,14 +938,19 @@ def get_osprofile(self):
         """Get the OS profile"""
         return VolatilityAPI(self.memfile).imageinfo()["data"][0]["osprofile"]
 
-    def run(self):
+    def run(self, manager=None, vm=None):
         results = {}
 
         # Exit if options were not loaded.
         if not self.voptions:
             return
 
-        vol = VolatilityAPI(self.memfile, self.osprofile)
+        # Check if theres a memory profile configured in the machinery config.
+        profile = Config(manager).get(vm).get("mem_profile")
```

Comment:
Use `config("%s:%s:mem_profile" % (manager, vm))` or perhaps some attribute from `self.task` if that's available (that'd be in the `Memory` class below, though), but I'd have to check that myself as well as I'm not sure if it's properly populated at the moment.